src/vl-form-message.js

```diff
@@ -46,6 +46,13 @@ export class VlFormLabel extends nativeVlElement(HTMLLabelElement) {
  * @see {@link https://webcomponenten.omgeving.vlaanderen.be/demo/vl-form-message.html|Demo}
  */
 export class VlFormValidationMessage extends nativeVlElement(HTMLParagraphElement) {
+  constructor() {
+    super();
+    if (this.hasAttribute('data-vl-error-id')) {
```

Comment:
code lijkt ok, ben enkel niet super mee waarom we 'data-vl-error-id' willen mappen / koppelen op het hidden attribuut. En vraag me ook af of we op deze native paragraaf component niet gewoon zelf hidden kunnen toggelen indien nodig. Nu voelt het wat alsof we native elementen aan het wrappen zijn.

Reply:
Net om te encapsuleren dat "de foutboodschap voor een bepaald input-veld altijd moet verborgen zijn als het inputveld geen error heeft en getoond moet worden als het input-veld wel een error heeft".

Uiteraard kan je dat ook in elke applicatie zelf oplossen, maar dan zou je een boel duplicatie hebben.